### PR TITLE
Highlight: Comment

### LINKED ISSUE #829: The ability to highlight comments?
image

### PR DESCRIPTION
**Pull request by HoshinoTented**
little work which resolves #829

### CODE REVIEW

File: cli/src/main/java/org/aya/cli/literate/SyntaxHighlight.java [hunk 1]
```diff
@@ -44,15 +44,22 @@ public class SyntaxHighlight implements StmtFolder<MutableList<HighlightInfo>> {
...
           return new HighlightInfo(sourcePos, type);
         });
-      semantics = semantics.concat(keywords);
+      var comments = tokens
+        .filter(x -> AyaParserDefinitionBase.SKIP_COMMENTS.contains(x.type()))
+        .map(t -> {
+          var sourcePos = AyaGKProducer.sourcePosOf(t, file);
+          var type = new HighlightInfo.SymLit(LitKind.Comment);
+          return new HighlightInfo(sourcePos, type);
+        });
```
Comment (imkiva) [inline cli/src/main/java/org/aya/cli/literate/SyntaxHighlight.java:61]: Maybe unify these with a new TokenSet.

**Review by imkiva — APPROVED** [review]

File: cli/src/main/java/org/aya/cli/literate/SyntaxHighlight.java (hunk 1 above)
Comment (HoshinoTented) [reply cli/src/main/java/org/aya/cli/literate/SyntaxHighlight.java:61]: Then `....contains(x.type())` to decide the highlight?

Comment (imkiva) [reply cli/src/main/java/org/aya/cli/literate/SyntaxHighlight.java:61]: Yes

**Comment by ice1000** [comment]
bors merge

**Pull request merged**